exercises/concept/ellens-alien-game/.docs/introduction.md

```diff
@@ -0,0 +1,104 @@
+# Introduction
+
+## Classes
+
+It is time to get to one of the core paradigms of C++: object-oriented programming (OOP).
+OOP is centered around `classes` - user-defined types of data with their own set of related functions.
+We will start with the basics and will cover more advanced topics further down the syllabus tree.
+
+### Members
+
+Classes can have **member variables** and **member functions**.
+They are accessed by the **member selection** operator `.`.
+Just as variables outside of `classes`, it is advisable to initialize member variables with a value upon declaration.
+This value will then become the default for newly created objects from this class.
+
+### Encapsulation and Information Hiding
+
+Classes offer the option to restrict access to their members.
+The two basic cases are `private` and `public`.
+`private` members are not accessible from outside the class.
+`public` members can be called freely.
+All members are `private` by default and need to be explicitly marked to be usable outside of the class.
```

Comment:
All members -> All members of a \`class\`
That might be helpful later on when we discuss the difference between struct and class

Also, I don't understand the second part of the sentence. It sounds like `private` members must be marked somehow to be accessible. Perhaps two separate sentences would be easier to understand.

Reply:
Do you think the new addition is safe for a later introduction of `protected`?